.github/workflows/git-actions.yml

```diff
@@ -25,7 +25,8 @@ jobs:
           flake8 . --count --show-source --statistics
       - name: Run unit tests
         run: |
+          export PYTHONPATH=${PWD}
```

Comment:
I found this: https://docs.pytest.org/en/stable/pythonpath.html#invoking-pytest-versus-python-m-pytest

So we could do: `python3 -m pytest` or just keep doing `PYTHONPATH=${PWD} pytest` (one line).

Reply:
The weird thing is that it works perfectly locally; I can't figure out why the Travis/GitHub `pytest` can't find the import. The environment shouldn't really be different once they've pulled the branch.

I did see that information, which suggests that the `python3 -m pytest` might work... but I can't test that locally because `pytest` doesn't fail locally, and it seemed silly to randomly push another change just to see what happens.

And, yeah; I did the `export` because I had another test command in the way, and I forgot to merge the path definition when I removed it.